Content.Server/_DV/Objectives/Systems/PickRandomTraitorSystem.cs

```diff
@@ -25,6 +25,7 @@ public override void Initialize()
     private void OnRandomTraitorAssigned(Entity<PickRandomTraitorComponent> ent, ref ObjectiveAssignedEvent args)
     {
         _pickTarget.AssignRandomTarget(ent, ref args, mindId =>
-            _role.MindHasRole<TraitorRoleComponent>(mindId));
+            _role.MindHasRole<TraitorRoleComponent>(mindId),
+            fallbackToAny: false); // bruh
```

Comment:
What happens when nobody's available now, does the objective just get ditched?

Reply:
yes its cancelled